angular/src/app/containers/map/_map.scss

```diff
@@ -0,0 +1,4 @@
+.c-map {
```

Comment:
In Web Components/Shadow DOM heb je de `:host` selector waarmee je het component zelf kunt stylen. Ik zou dit dan ook altijd gebruiken omdat het conform de nieuwe standaarden is. Daarbij is het heel consistent en ben je gelijk van die BEM namespacing af.

EDIT: Blijkbaar betreft dit een directive, ik weet niet 100% zeker of `:host` dan ook werkt.